src/portals/hope/features/dashboard/TasksData.tsx

```diff
@@ -82,6 +87,9 @@ const AreaBox = styled.div`
     }
   }
 `
+const CleanfilterButton = styled(AreaBox)`
```

Comment:
```suggestion
const ClearFilterButton = styled(AreaBox)`
```
Might be a better name? :)